lib/openid_connect/protected_resource_endpoint.rb

```diff
@@ -0,0 +1,29 @@
+module OpenidConnect
+  module ProtectedResourceEndpoint
+    def self.included(klass)
+      klass.send :include, ProtectedResourceEndpoint::Helper
+    end
+
+    module Helper
+      def current_token
+        @current_token
+      end
+    end
```

Comment:
Why this pattern? why not just an `attr_reader :current_token` in the module directly?